src/transformers/models/musicgen/modeling_musicgen.py

```diff
@@ -545,7 +545,6 @@ def _upad_input(self, query_layer, key_layer, value_layer, attention_mask, query
         )
 
 
-# Copied from transformers.models.bart.modeling_bart.BartSdpaAttention with Bart->Musicgen
```

Comment:
I removed the statement here, just want to make sure that it's okay for you @amyeroberts !

Reply:
Yep!